libs/blocks/bulk-publish/bulk-publish.js

```diff
@@ -87,6 +87,15 @@ function bulkPreviewStatus(row) {
   `;
 }
 
+function bulkIndexStatus(row) {
+  const status = !(row.status.index === 200 || row.status.index === 202)
+    ? `Error - Status: ${row.status.index}`
+    : '';
+  return !(row.status.index === 200 || row.status.index === 202) && row.status.index !== undefined && html`
```

Comment:
nit: assign `!(row.status.index === 200 || row.status.index === 202)` to a const